javascript-bank/account.js

```diff
@@ -0,0 +1,41 @@
+/* exported Account */
+
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype = {
```

Comment:
It's generally not a good idea to re-assign a new object to a constructor's `prototype` if you don't have to, as it breaks the existing prototype chain (including the built-in `constructor` property). Instead, add the custom methods to the existing `prototype`.

---

![image](https://user-images.githubusercontent.com/16143771/123344587-07f74680-d509-11eb-9df8-c89171313aba.png)

Reply:
fixed, but same screenshot still lol